pyproject.toml

```diff
@@ -34,9 +35,18 @@ dependencies = [
 # On a mac, install optional dependencies with `pip install '.[dev]'` (include the single quotes)
 [project.optional-dependencies]
 dev = [
+    "ipykernel", # Support for Jupyter notebooks
+    "ipython", # Also used in building notebooks into Sphinx
+    "matplotlib", # Used in sample notebook intro_notebook.ipynb
```

Comment:
```suggestion
```
Already listed in dependencies (& we aren't including intro_notebook.ipynb anyway)